xds/googledirectpath/googlec2p.go

```diff
@@ -58,15 +59,68 @@ const (
 
 // For overriding in unittests.
 var (
-	onGCE   = googlecloud.OnGCE
-	randInt = rand.Int
-	logger  = internalgrpclog.NewPrefixLogger(grpclog.Component("directpath"), logPrefix)
+	onGCE            = googlecloud.OnGCE
+	randInt          = rand.Int
+	logger           = internalgrpclog.NewPrefixLogger(grpclog.Component("directpath"), logPrefix)
+	universeDomainMu sync.Mutex
+	universeDomain   = ""
 )
 
 func init() {
 	resolver.Register(c2pResolverBuilder{})
 }
 
+// SetUniverseDomain informs the gRPC library of the TPC universe domain
+// in which the process is running. It is the caller's responsibility to
```

Comment:
Is there any documentation we can point to about this?

Reply:
Asking around, I'll get back to this comment after I hear back

Reply:
No documentation currently